docs/system-requirements.md

```diff
@@ -0,0 +1,44 @@
+# System requirements
+
+## Server
+Additional to normal Nextcloud requirements the following constraints apply to use Nextcloud Talk:
+
+### HTTPS
+ 
+HTTPS is required to be able to use WebRTC (the video call technic of browsers used by Nextcloud Talk calls).
+
+### Database
+* SQLite: must not be used, to grant a decent experience for chats and calls
+* MySQL/Maria DB: Must enable utf8mb4 support** as per documentation at [Enabling MySQL 4-byte support](https://docs.nextcloud.com/server/latest/admin_manual/configuration_database/mysql_4byte_support.html)
+
+### Webserver
+
+Apache and Nginx must use:
+
+* PHP FPM + mpm_events or
+* PHP + mpm_prefork
+
+Other combinations will not work due to the long polling used for chat and signaling messages, see [this issue](https://github.com/nextcloud/spreed/issues/2211#issuecomment-610198026) for details.
+
+### TURN server
+
+A TURN server running on **port 443** (or 80) is required in almost all scenarios, see  [Configuring coTURN](TURN.md) for more details.
```

Comment:
This reads like one always needs a TURN server. In my experience a TURN server isn't always required.

Reply:
Well as soon as a mobile phone is involved you need one 99% of the time. If people want to ignore it fine by me, but at least we told them up front. The UI will tell it again later when you try a call and a connection can not be established.